x-pack/plugins/enterprise_search/public/applications/workplace_search/__mocks__/content_sources.mock.ts

```diff
@@ -314,14 +315,18 @@ export const exampleResult = {
       { fieldName: 'dogs', label: 'Canines' },
     ],
   },
-  titleFieldHover: false,
-  urlFieldHover: false,
```

Comment:
These two do not exist in a server response and TS is happy with the removal. Let me know if I should keep them.